interactive_engine/src/executor/store/src/v2/api/mod.rs

```diff
@@ -19,15 +19,20 @@ mod partition_snapshot;
 mod types;
 mod condition;
 
+pub use self::partition_snapshot::*;
+pub use self::types::*;
+pub use self::condition::*;
+
 pub type SnapshotId = u64;
 pub type LabelId = u32;
 pub type PropertyId = u32;
 pub type VertexId = u64;
 pub type EdgeInnerId = u64;
 pub type SerialId = usize;
 
-type Records<T> = Box<dyn Iterator<Item=Result<T>> + Send>;
+pub type Records<T> = Box<dyn Iterator<Item=T> + Send>;
```

Comment:
What if iterator.next() returns Error?

Reply:
I think that if error occurs, it will return None.

Reply:
If caller get None, they cannot distinguish error from the end of iterator.

Reply:
make sense